src/base.jl

```diff
@@ -496,21 +496,57 @@ Initialise then begin a least squares fitting of the supplied data.
 - `singularity`: Presence of singularity in model
 - `indweight`: indices weighting, see `modelfit` for more discussion
 - `optmethod`: optimisation algorithm used by NLOpt. 
-"""
-function leastsquares_init(params_init::Vector{RheoFloat}, low_bounds::RheovecOrNone,
-                           hi_bounds::RheovecOrNone, modulus,
-                           time_series::Vector{RheoFloat}, dt::RheoFloat,
-                           prescribed_dot::Vector{RheoFloat}, measured::Vector{RheoFloat};
-                           insight::Bool = false, constant_sampling::Bool=true,
-                           singularity::Bool = false, _rel_tol = 1e-4, indweights=nothing,
-                           optmethod::Symbol = :LN_SBPLX)
+- `opttimeout`: allows user to set a wall clock timeout on optimisation 
+"""
+function leastsquares_init(params_init::Vector{RheoFloat},
+                            low_bounds::RheovecOrNone,
+                            hi_bounds::RheovecOrNone, 
+                            modulus,
+                            time_series::Vector{RheoFloat},
+                            dt::RheoFloat,
+                            prescribed_dot::Vector{RheoFloat},
+                            measured::Vector{RheoFloat};
+                            insight::Bool = false,
+                            constant_sampling::Bool=true,
+                            singularity::Bool = false,
+                            rel_tol::Union{Real,Nothing} = nothing,
```

Comment:
should we actually rename this `tol_rel_x` for consistency?

Reply:
Yes, I just left it as-is, but it makes more sense to name it specifically.